js/interface.js

```diff
@@ -813,7 +813,7 @@ function browseDataSource(id) {
   ]).then(function() {
     windowResized();
 
-    if (widgetData.view === 'access-rules') {
+    if (widgetData.view === 'access-rules' && widgetData.context !== 'overlay') {
```

Comment:
Add comment to explain why `!== 'overlay'` is added

Reply:
to resolve UI glitch issue we need to add this.

Reply:
@amolsuryawanshi47 This is not a technical explanation. And I need comment added into the code as inline comment with `//`

Reply:
@tonytlwu  as per your comment, this scenarios came in picture [here](https://github.com/Fliplet/fliplet-widget-data-source/blob/master/js/interface.js#L840-L842), Once we click on _Mange security rules_ button. So now I realized below code is not required because afterwards. I have added same condition for this scenario in different place. And due to this condition DS security rules table glitch issue is coming. So removing below code in this PR. We will give it to QA for testing.

` if (widgetData.view === 'access-rules' && widgetData.context !== 'overlay')  {
  $('#show-access-rules').click();
}`

Cc: @NikitaJadhavXoriant

Reply:
@amolsuryawanshi47 

> I have added same condition for this scenario in different place.

Where?

Reply:
@tonytlwu we are adding [here](https://github.com/Fliplet/fliplet-widget-data-source/blob/master/js/interface.js#L838-L842) and we are checking only `if (widgetData.view === 'access-rules')` condition.

Reply:
👍🏻